### ISSUE TITLE
solicit AFSCME Local 1624 feedback re: Point 2

### ISSUE THREAD

**Issue by lianilychee** [open-austin/ai-policy#16]
## Prerequisites
- [ ] #1 

## Notes
EFF's feedback on our one-pager:

> *Consider long term economic effects now* && *Be accountable to everyone, equally.*
These sections are out of scope for EFF, so can't give much feedback.
As an idea though, check in with local public workers union AFSCME local 1624. Other labor groups have called for notice and rights to appeal decisions, which I think is a bit more than what is asked here.

**Comment by ReadDj** [comment]
Seemed like our conversation needs to be longer on this one.  I believe we'll talk more on this in smaller group unless others want to join the convo.

**Comment by lianilychee** [comment]
@ReadDj agree.  I think you are right, that we need to make the toolkit first.  So I indcated making hte toolkit as a pre-req to this task, and also moved its status all the way back to no status
